add readme for flax clm (#12111)
huggingface/transformers: examples/flax/language-modeling/README.md

For a step-by-step walkthrough of how to do masked language modeling in Flax, please have a 
look at [this TODO: (Patrick)]() google colab.

## Causal language modeling

> In the following, we demonstrate how to train a auto-regressive causal transformer model 

Reviewer: ```suggestion
In the following, we demonstrate how to train an auto-regressive causal transformer model 
```

> as introduced in [GPT2](https://openai.com/blog/better-language-models/).

Reviewer: ```suggestion
in JAX/Flax.
```

> More specifically, we demonstrate how JAX/Flax can be leveraged 

Reviewer: ```suggestion
More specifically, we pretrain a randomely initialized [**`gpt2`**](https://huggingface.co/gpt2) model in Norwegian on a single TPUv3-8.
```

to pre-train 124M [**`gpt2`**](https://huggingface.co/gpt2)
in Norwegian on a single TPUv3-8 pod.

The example script uses the 🤗 Datasets library. You can easily customize them to your needs if you need extra processing on your datasets.

Let's start by creating a folder to save the trained model and a symbolic link to the `run_clm_flax.py` script.

```bash
export MODEL_DIR="./norwegian-gpt2"
mkdir -p ${MODEL_DIR}
ln -s ~/transformers/examples/flax/language-modeling/run_clm_flax.py run_clm_flax.py
```

> Netx, we'll follow the similar steps to train the tokenizer as given in [Train tokenizer](#train-tokenizer)

Reviewer: ```suggestion
Next, we'll follow the same steps as above in [Train tokenizer](#train-tokenizer) to train the tokenizer.
```
